schema/project-level/project-schema.json

```diff
@@ -477,6 +477,19 @@
               "title": "Administrative entity",
               "description": "The name and identifier of the entity responsible for contract administration if this is different from the procuring entity. The full details of the entity should be added to the project-level `parties` array with a `role` of 'administrativeEntity'.",
               "$ref": "#/definitions/OrganizationReference"
+            },
+            "sustainability": {
+              "title": "Sustainability",
+              "description": "Information about how the contracting process incorporates sustainable public procurement.",
+              "type": [
+                "array"
+              ],
+              "items": {
+                "$ref": "#/definitions/Sustainability"
+              },
+              "wholeListMerge": true,
```

Comment:
OC4IDS has no merging mechanism.

```suggestion
```

Reply:
I remember now why I added this. It's because without it the tests fail due to there being no `id` field in `sustainability`. There's no `id.sustainability` in the OCDS extension, but it does have the `wholeListMerge` property.

So either we add the `id` in which makes it diverge from the OCDS extension, or we keep in the `wholeListMerge` property even though it doesn't actually have any meaning in OC4IDS

Reply:
Hmm, I think we might need to update the tests in that case. @jpmckinney what do you think?